gazebo_ros/launch/spawn_entity_demo.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
```

Comment:
roslaunch is not finding this file for me. If I rename it to `spawn_entity_demo.launch.py` the command line autocompletes and the file can be launched.